### DISCUSSION TITLE
openssl: error while loading shared libraries: libssl.so.3: cannot open shared object file: No such file or directory

### DISCUSSION THREAD

**Discussion by tgupta2222** [openssl/openssl#21496] [Q&A]
I have debian 8 (Jessie) and had installed openssl 1.0.1t version.

Now, with regards to some software installation when I tried to upgrade the openssl version to >3.0 it caused issue. I uninstalled and am not sure if it did. 

Now, when I try to find the openssl version it gave the error: openssl: error while loading shared libraries: libssl.so.3: cannot open shared object file: No such file or directory

I have newly installed openssl version 1.0.1t but in vain. It is failing to load any openssl version. I am not sure if it will destabilise the linux debian system.

I tried all possible solutions from suggested over the web resources, but that did not help. Please  suggest remedy

**Comment by t8m** [comment]
This is a debian-specific question. It seems you still have the openssl binary executable from the 3.0 installation but the libraries are removed. How did you install the 3.0 version? Where is the binary placed (use `which openssl`)?

**Reply by tgupta2222** [reply]
I am sorry. I am completely new to this. 
upon which openssl, I found : /usr/local/bin/openssl 
And I could see that the openssl located here is not updated with my latest installation.
How can I put the latest one in this location /usr/local/bin/openssl ? 
Please suggest.

**Reply by tgupta2222** [reply]
Thanks I just fixed this issue. 
I am wondering for Debian8, which openssl version is most suitable?
